### PR TITLE
fix: search input layout animation

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

### Description

- Fix buggy search input animation
- This serves as an example on how we might solve some of the issues here https://artsyproduct.atlassian.net/browse/CX-1096

**Before**

https://user-images.githubusercontent.com/11945712/111333117-a7877c00-8672-11eb-8d12-7dfb71d71e01.mov

___
**After**

https://user-images.githubusercontent.com/11945712/111333108-a5bdb880-8672-11eb-922f-dd9055e6b66a.mov

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434

### CODE REVIEW

File: src/lib/Components/SearchInput.tsx [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 interface SearchInputProps extends InputProps {
   enableCancelButton?: boolean
 }
 
+if (Platform.OS === "android") {
+  if (UIManager.setLayoutAnimationEnabledExperimental) {
+    UIManager.setLayoutAnimationEnabledExperimental(true)
+  }
+}
+
```
Comment (ds300) [inline src/lib/Components/SearchInput.tsx:16]: should we just set this globally in `index.android.js`?

Comment (MounirDhahri) [reply src/lib/Components/SearchInput.tsx:16]: yes, yes, yes

Comment (MounirDhahri) [reply src/lib/Components/SearchInput.tsx:16]: I will set it inside AndroidApp.tsx

**Comment by ds300** [comment]
great catch! ❤️

**Pull request merged**